utils/swift_build_support/swift_build_support/products/wasisysroot.py

```diff
@@ -195,8 +195,8 @@ def _build(self, host_target, enable_wasi_threads, compiler_rt_os_dir, target_tr
         if enable_wasi_threads:
             c_flags.append('-pthread')
             cxx_flags.append('-pthread')
-        cmake.cmake_options.define('CMAKE_C_FLAGS:STRING', ' '.join(c_flags))
-        cmake.cmake_options.define('CMAKE_CXX_FLAGS:STRING', ' '.join(cxx_flags))
+        cmake.cmake_options.define('CMAKE_C_FLAGS', ' '.join(c_flags))
```

Comment:
Could you elaborate on how the type specifier makes difference? Why is this specific to `CMAKE_C{,XX}_FLAGS`?

Reply:
Disregard that, it's caused by `CMAKE_CXX_FLAGS` defined on macOS preset, which overrides these `CMAKE_CXX_FLAGS`, I'll clean that up.